### PR TITLE
Feature/find student package

### PR DESCRIPTION
**Pull request by millerts**
Added a new student endpoint that will fetch the fully populated student from the XML package.  This will be leveraged by the examinee information we're planning to store in the exam db.

I purposefully left the existing endpoints and code for backwards compatibility but I imagine we'll remove the fetchStudentById in the near term.

I also removed the Service integration tests since we don't do that anymore.

### CODE REVIEW

File: client/src/test/java/tds/student/StudentTest.java [hunk 1]
```diff
@@ -1,18 +0,0 @@
-package tds.student;
-
-import org.junit.Test;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class StudentTest {
-
-    @Test
-    public void shouldBeCreated() {
```
Comment (jtreuting) [inline client/src/test/java/tds/student/StudentTest.java]: Is there a reason this got removed?  Shouldn't this test the builder.

File: service/src/main/java/tds/student/services/impl/RtsServiceImpl.java [hunk 2]
```diff
@@ -50,4 +56,137 @@ public RtsServiceImpl(RtsStudentPackageQueryRepository rtsStudentPackageQueryRep
...
+        attributes.add(new RtsStudentPackageAttribute("HispanicOrLatinoEthnicity", student.getHispanicOrLatinoEthnicity()));
+        attributes.add(new RtsStudentPackageAttribute("AmericanIndianOrAlaskaNative", student.getAmericanIndianOrAlaskaNative()));
+        attributes.add(new RtsStudentPackageAttribute("Asian", student.getAsian()));
+        attributes.add(new RtsStudentPackageAttribute("BlackOrAfricanAmerican", student.getBlackOrAfricanAmerican()));
+        attributes.add(new RtsStudentPackageAttribute("White", student.getWhite()));
+        attributes.add(new RtsStudentPackageAttribute("NativeHawaiianOrOtherPacificIslander", student.getNativeHawaiianOrOtherPacificIslander()));
+        attributes.add(new RtsStudentPackageAttribute("DemographicRaceTwoOrMoreRaces", student.getDemographicRaceTwoOrMoreRaces()));
+        attributes.add(new RtsStudentPackageAttribute("IDEAIndicator", student.getIDEAIndicator()));
+        attributes.add(new RtsStudentPackageAttribute("LEPStatus", student.getLEPStatus()));
+        attributes.add(new RtsStudentPackageAttribute("Section504Status", student.getSection504Status()));// TODO: UNKNOWN/CANNOT PROVIDE
```
Comment (jtreuting) [inline service/src/main/java/tds/student/services/impl/RtsServiceImpl.java]: Is this TODO for us or left over from other code?

**Review by jtreuting — APPROVED** [review]

File: service/src/main/java/tds/student/services/impl/RtsServiceImpl.java (hunk 2 above)
Comment (millerts) [reply service/src/main/java/tds/student/services/impl/RtsServiceImpl.java]: It was a straight copy from the existing code.  Removed.

File: client/src/test/java/tds/student/StudentTest.java (hunk 1 above)
Comment (millerts) [reply client/src/test/java/tds/student/StudentTest.java]: I removed it a bit out of laziness during the refactor as I was modifying it quite a bit.  I've added the test back in verifying the builder works as expected.

**Pull request merged**